plugins/scaffolder-backend/src/scaffolder/stages/publish/helpers.ts

```diff
@@ -35,16 +35,16 @@ export async function initRepoAndPush({
     logger,
   });
 
-  await git.init({
-    dir,
-  });
-
   const paths = await globby(['./**', './**/.*'], {
     cwd: dir,
     gitignore: true,
     dot: true,
   });
 
+  await git.init({
```

Comment:
This change is not required anymore. I've just merged a fix for this.

Reply:
done